install.go

```diff
@@ -106,8 +106,6 @@ func install(parser *arguments) error {
 			arguments.options["ignore"] = ignoreStr
 		}
 
-		fmt.Println()
```

Comment:
I'm not sure about this. Does it affect anything else?

Reply:
There's a couple of new lines sprinkled over to make things read better. What each one does exactly I can not remember.

Reply:
I'm looking at the 75th line and think this empty line only for upgrades.

I like readability, including via empty lines.

But `pacman` doesn't have them. And, I think, if we want to make `yay` more native with `pacman` — we should make output more similar too.

At least, the "jumping" empty line, depending on the `combinedupgrade` option, is strange (for me).

Reply:
I think you're confusing the native pacman wiki criteria with looking like pacman. The former is all about how AUR helpers interact with pacman and don't do dangerous things.

Personally I don't want Yay to mimic pacman's look too much. Sure I want to make sure it seems familiar but I also want to remind users that this is a different application.

But back on topic the empty line does not depend on `combinedupgrade`. It's applied either way. But yes only the upgrade menu has this line while the clean/diff/edit do not. So just for the sake of consistency I say remove it.

Reply:
> I think you're confusing the native pacman wiki criteria with looking like pacman. The former is all about how AUR helpers interact with pacman and don't do dangerous things.

No, I understand.

> Personally I don't want Yay to mimic pacman's look too much. Sure I want to make sure it seems familiar but I also want to remind users that this is a different application.

1. I like versions diff output by `yay`, and I don't want to see this super-simple output from `pacman`.
2. Just my eye clings at the same phrase (sentence) with a slightly different spelling.

> But back on topic the empty line does not depend on combinedupgrade. It's applied either way.

Yes, so I mentioned "jumping" empty line. There is it. But at different places. Also a bit strange (looks like a bug) for me.